fabfile.py

```diff
@@ -844,6 +848,111 @@ def updatemaincode(backup=False):
 
             venvcmd_py3('pip install -r ../url_metadata/requirements.txt')
 
+        if env.uses_bluenove_actionable:
+            execute(update_bluenove_actionable)
```

Comment:
We may not want to rebuild the docker image each time we deploy. What time does it take to rebuild the image? I'll remove those lines. @ayyazdaniaryan what do you think?

Reply:
Yes we can do that manually!